tests/cpp/operator/tune/operator_tune_test.cc

```diff
@@ -18,13 +18,14 @@
  */
 #include <gtest/gtest.h>
 #include <mxnet/tensor_blob.h>
-#include <numeric>
 #include "../../src/operator/nn/activation-inl.h"
 #include "../../src/operator/operator_tune-inl.h"
 #include "../include/test_op_runner.h"
 #include "../include/test_core_op.h"
 #include "../include/test_tune.h"
 
+#if MXNET_USE_OPERATOR_TUNING
```

Comment:
Shouldn't a test always run except if it is unsupported on some platforms? Is there any reason operator tuning could not be done on some platforms? I'd prefer to see this removed.

Reply:
Operator tuning is an optional feature. It also hangs on startup on Windows for some reason so that's disabled in the CMAKE.

Reply:
Shouldn't this issue be investigated? I mean, this sounds like a kind of serious problem.